### ISSUE TITLE
A11y_Keyboard focus is not visible on 'Pip' package while navigating with tab key_Accessibility Testing for Python in VS_Python Environments_Keyboard 

### ISSUE THREAD

**Issue by cwebster-99** [microsoft/PTVS#7641]
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1835769
